project_national/housing_characteristics/HVAC Secondary Heating Fraction.tsv

```diff
@@ -0,0 +1,3 @@
+Option=None	Option=10% Served	Option=20% Served	Option=30% Served	Option=40% Served	Option=50% Served	sampling_probability
```

Comment:
I'd suggest naming this file something like "HVAC Secondary Heating Partial Space Condition.tsv" so that it is consistent with the existing "HVAC Cooling Partial Space Conditioning.tsv" file name.